springdoc-openapi-starter-common/src/main/java/org/springdoc/core/utils/SchemaUtils.java

```diff
@@ -189,6 +190,9 @@ public static boolean fieldRequired(Field field, @Nullable io.swagger.v3.oas.ann
 	public static void applyValidationsToSchema(Schema<?> schema, List<Annotation> annotations) {
 		annotations.forEach(anno -> {
 			String annotationName = anno.annotationType().getSimpleName();
+			if (annotationName.equals(Positive.class.getSimpleName())) {
+				schema.setMinimum(BigDecimal.ONE);
```

Comment:
Value `BigDecimal.ONE` is valid only for integers.
For decimals, for example, we can have a valid value `0.00001`.